pkg/api/schema/schema.proto

```diff
@@ -349,24 +352,54 @@ message VerifiableZAddRequest {
 }
 
 message TxRequest {
-	uint64 tx = 1;
+	uint64      tx = 1;
+	EntriesSpec entriesSpec = 2;
+	uint64 sinceTx  = 3;
+	bool   noWait   = 4;
+}
+
+message EntriesSpec {
+	EntryTypeSpec kvEntriesSpec  = 1;
+	EntryTypeSpec zEntriesSpec   = 2;
+	EntryTypeSpec sqlEntriesSpec = 3;
+}
+
+message EntryTypeSpec {
+	EntryTypeAction action = 1;
+}
+
+enum EntryTypeAction {
+	EXCLUDE     = 0;
+	ONLY_DIGEST = 1;
+	RAW_VALUE   = 2;
```

Comment:
how about only the referenced key info? like I want to get parsed reference but not query for the referenced value (this of course could be a future improvement)

Reply:
yes, we can add it later on